ratpack-groovy/src/main/java/ratpack/groovy/Groovy.java

```diff
@@ -141,6 +141,31 @@ public static void ratpack(@DelegatesTo(value = Ratpack.class, strategy = Closur
      */
     void serverConfig(@DelegatesTo(value = ServerConfigBuilder.class, strategy = Closure.DELEGATE_FIRST) Closure<?> configurer);
 
+    /**
+     * Evaluates the provided path using the Ratpack DSL and applies the configuration to this server.
+     * <p>
+     * The configuration supplied by the included path are applied inline with the existing parent configuration.
+     * This allows the same semantics as specifying the configuration in a single file to be followed.
+     * For {@link Ratpack#bindings(Closure)}, the configuration is appended.
+     * For {@link Ratpack#handlers} and {@link Ratpack#serverConfig(Closure)}, the configuration is merged.
+     * Settings from the parent configuration that are applied after the {@code include}, will be applied after the child configurations.
+     *
+     * @param path The absolute path to the external Groovy DSL to included into the server.
+     * @since 1.3
+     */
+    void include(Path path);
```

Comment:
We should mention here that this should only be called at the top level. We should also enforce this by failing if it is called during handlers {} etc.